db-commons/src/main/resources/reference.conf

```diff
@@ -2,6 +2,36 @@ bitcoin-s {
     datadir = ${HOME}/.bitcoin-s
     network = regtest # regtest, testnet3, mainnet
 
+    logging {
+        level = info # trace, debug, info, warn, error, off
+
+        # You can also tune specific module loggers.
+        # They each take the same levels as above.
+        # The available loggers are: 
```

Comment:
can you give an example? For instance if i wanted to tune p2p would i do 

```
logging.p2p = INFO
```
?